client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.ui.xml

```diff
@@ -51,7 +51,7 @@
                                     <i ui:field="cartIcon" class="{font.icons.iconCart}"/>
                                     <span class="{res.style.visuallyHidden}">cart</span>
                                 </button>
-                                <div class="{res.style.cart_notifications}">
+                                <div ui:field="tooltip" class="{res.style.cart_notifications}">
```

Comment:
numberOfItemsTooltip (`tooltip` is confusing since ApplicationView contains the whole app).